.github/workflows/Pkg EdFi.Ods.CodeGen.yml

```diff
@@ -59,20 +60,6 @@ jobs:
         .\build.githubactions.ps1 build -Configuration ${{ env.CONFIGURATION }} -InformationalVersion ${{ env.INFORMATIONAL_VERSION}} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -Solution "$env:GITHUB_WORKSPACE/Ed-Fi-ODS/Utilities/CodeGeneration/EdFi.Ods.CodeGen.sln"
       shell: pwsh
 
-    - name: Run Unit tests for  codegen Project
-      working-directory: ./Ed-Fi-ODS/
-      run: |
-        $ErrorActionPreference = 'Stop'
-        $solution = "$env:GITHUB_WORKSPACE\Ed-Fi-ODS\Utilities\CodeGeneration\EdFi.Ods.CodeGen.sln"
-        $reports = "$env:GITHUB_WORKSPACE\Ed-Fi-ODS-Implementation\reports\"
-        if (Test-Path $reports) {
-            Remove-Item -Path $reports -Force -Recurse | Out-Null
-        }
-        New-Item -ItemType Directory -Force -Path $reports | Out-Null
-        $reportName = $reports + "EdFi.Ods.CodeGen.sln.xml"
-        & dotnet test $solution --configuration ${{ env.CONFIGURATION }} --filter TestCategory!=LocalOnly --logger "trx;LogFileName=$reportName"
-      shell: pwsh
```

Comment:
The task for creating a new NuGet package for code generation [failed](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7715262745/job/21029397326) due to two of the approval tests. I don't think we should be running the approval tests here though, and we're already running unit and integration tests in the separate pull request workflow.